src/cmd-coreos-prune

```diff
@@ -172,8 +172,7 @@ def main():
                             prune_images(s3_client, current_build, images_to_keep, args.dry_run, bucket, prefix)
                         # Fully prune releases that are very old including deleting the directory in s3 for that build.
                         case "build":
-                            prune_build(s3_client, bucket, prefix, build_id, args.dry_run)
-                            pruned_build_ids.append(build_id)
+                            builds_to_prune.append(build_id)
```

Comment:
```suggestion
                            # Since pruning a build prunes s3 for all architectures
                            # we'll prune later so that we do it only once and we
                            # make sure we've completed any architecture specific
                            # operations, (i.e. pruning aarch64 AMIs).
                            builds_to_prune.append(build_id)
```